src/configuration.ts

```diff
@@ -582,9 +586,10 @@ export function getCommandForConfiguration(configuration: string | undefined): v
 
         // If configuration command has a path (absolute or relative), check if it exists on disk and error if not.
         // If no path is given to the make tool, search all paths in the environment and error if make is not on the path.
+        const notFoundStr: string = localize("not.found", "not found.");
         if (configurationCommandPath  !== "") {
             if (!util.checkFileExistsSync(configurationMakeCommand)) {
-                vscode.window.showErrorMessage("Make not found.");
+                vscode.window.showErrorMessage(`Make ${notFoundStr}.`);
```

Comment:
I would use `{0} not found.`.  The more contents you can give the loc team, the better.